src/flash/flashCmd.ts

```diff
@@ -103,3 +124,206 @@ export async function verifyCanFlash(
   }
   return continueFlag;
 }
+
+export function isFlashEncryptionEnabled(workspaceRoot: vscode.Uri) {
+  const flashEncryption = utils.getConfigValueFromSDKConfig(
```

Comment:
I suggest putting these flash verification commands in a separate file, something like `flash/verifyFlashEncryption.ts`